cli/azd/cmd/env.go

```diff
@@ -26,19 +26,17 @@ import (
 )
 
 func envActions(root *actions.ActionDescriptor) *actions.ActionDescriptor {
-	envCmd := &cobra.Command{
-		Use:   "env",
-		Short: "Manage environments.",
-		//nolint:lll
-		Long: `Manage environments.
-
-With this command group, you can create a new environment or get, set, and list your app environments. An app can have multiple environments (for example, dev, test, prod), each with a different configuration (that is, connectivity information) for accessing Azure resources.
-
-You can find all environment configurations under the ` + output.WithBackticks(`.azure\<environment-name>`) + ` directories. The environment name is stored as the AZURE_ENV_NAME environment variable in the ` + output.WithBackticks(`.azure\<environment-name>\directory\.env`) + ` file.`,
-	}
-
 	group := root.Add("env", &actions.ActionDescriptorOptions{
-		Command: envCmd,
+		Command: &cobra.Command{
+			Use:   "env",
+			Short: "Manage environments.",
+		},
+		HelpOptions: actions.ActionHelpOptions{
+			Description: getCmdEnvHelpDescription,
+		},
+		GroupingOptions: actions.CommandGroupOptions{
+			RootLevelHelp: actions.CmdGroupManage,
+		},
```

Comment:
Are we introducing too many concepts here? Why is the `Help` text separate than the `cobra.Command`? I think in the old world, we would set `Long` under `cobra.Command`, can we get back to that world? What if we did:

```golang
type command struct {
*cobra.Command
Description string // or is this `Long`?
Footer string // or is this `Examples`?
}
```

and have this one struct that defines the command design?

Reply:
> Why is the Help text separate than the cobra.Command

Because cobra produces the help text by using the `Long`,  `Usage` and calling `Flags.Usage`.  The way we tell cobra to use a different format is by setting the `HelpTemplate`.

Then, the `HelpTemplate` can't be set until the command is built (all sub-commands are added to the root node). And that happen until the action descriptor is used to produce the command (binding flags and subcommands to the root node).

That's why I append the `HelpOptions` to the actions descriptor, as that's what we use to describe and build the command.